aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/AuthEnvironment.kt

```diff
@@ -33,14 +34,50 @@ import java.util.Date
 import java.util.UUID
 
 internal class AuthEnvironment internal constructor(
+    val context: Context,
     val configuration: AuthConfiguration,
     val cognitoAuthService: AWSCognitoAuthServiceBehavior,
     val credentialStoreClient: StoreClientBehavior,
     private val userContextDataProvider: UserContextDataProvider? = null,
     val hostedUIClient: HostedUIClient?,
     val logger: Logger
 ) : Environment {
+
+    companion object {
+        /*
+        Auth plugin needs to read from Pinpoint shared preferences, but we don't currently have an architecture
```

Comment:
Credential store is supposed to be the tool that allows plugins to share data.

Reply:
This is not a credential so we should look into possibly renaming, if that is the case.